client/my-sites/domains/domain-management/settings/index.tsx

```diff
@@ -36,6 +36,7 @@ import {
 	hasLoadedSitePurchasesFromServer,
 } from 'calypso/state/purchases/selectors';
 import { getCurrentRoute } from 'calypso/state/selectors/get-current-route';
+import getSites from 'calypso/state/selectors/get-sites';
```

Comment:
This is memoized but we should consider if the filtering logic will impact performance when checking all of the user's sites. It's going to run every time someone opens the Settings page for a domain. We could add another memoized selector that includes the filtering like https://github.com/Automattic/wp-calypso/blob/241c15775176d098b866a8c72b025bdbe0cbedce/client/state/selectors/can-any-site-have-plugins.ts

Reply:
That's great, how `can-any-site-is-connectable` sounds to you?

Reply:
Great! So we're checking: are there any sites we can connect a domain to. 🎉